packages/slice-machine/templates/slice/gatsby/index.js

```diff
@@ -0,0 +1,30 @@
+// This file was generated by Slice Machine. You may edit this file.
+
+import * as React from 'react'
+import { graphql } from 'gatsby'
+
+// TODO: Implement the {{componentName}}Slice component.
+//
+// → Learn how to render this component with <SliceZone>
+//   https://prismic.io/docs/technologies/template-slices-gatsby
+export default function {{componentName}}Slice({ slice }) {
+  return (
+    <section>
```

Comment:
I see that you generate a model with a **title** and a **description** field but those fields are not use in the template component. Is it something intended ?

Reply:
Yes, this is intentional. Gatsby is unique in that it requires a GraphQL query to fetch data. If the query contains invalid fields, the project fails to start or build.

Since `title` and `description` are default model fields, they are very likely to be deleted immediately by the developer. Since they may be deleted, it means the Gatsby project would quickly be in a broken state. The developer would need to go into the component and edit the query and component before moving forward.

By keeping `title` and `description` out of the component and GraphQL query, it will automatically support any immediate changes the developer makes to their model within Slice Machine.

Hope that makes sense! Let me know if I should describe in more detail. :)